docs/source/use_cases/face_mask_detection.md

```diff
@@ -0,0 +1,47 @@
+# Face Mask Detection
+
+## Overview
+
+As part of COVID-19 measures, the Singapore Government has mandated the wearing of face masks in public places. AI Singapore has developed a solution that checks whether or not a person is wearing a face mask. This can be used in places such as in malls or shops to ensure that visitors adhere to the guidelines.
+
+We have trained a custom Yolov4 model to detect whether or not a person is wearing a face mask. This is further elaborated in a [subsequent section](#how-it-works).
+
+## Demo
+
+To try our solution on your own computer with [PeekingDuck installed](../getting_started/01_installation.md): use the following configuration file: [face_mask_detection.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/face_mask_detection.yml) and run PeekingDuck.
+
+```
+> peekingduck run --config_path <path_to_face_mask_detection.yml>
+```
+
+## How it Works
+
+The main component is the detection of face mask using the custom Yolov4 model.
+
+**Face Mask Detection**
+
+We use an open source object detection model known as [Yolov4](https://arxiv.org/abs/2004.10934) and its smaller and faster variant known as Yolov4-tiny to identify the bounding boxes of human faces with and without face masks. This allows the application to identify the class (no_mask =0; mask = 1) and the location of faces in the video feed. The location is returned as two (x, y) coordinates in the form [x1, y1, x2, y2], where (x1, y1) is the top-left corner of the bounding box, and (x2, y2) is the bottom-right. These are used to form the bounding box of each human face detected. For more information in how adjust the yolo face node, checkout the [Yolo face configurable parameters](/peekingduck.pipeline.nodes.model.yolo_face.Node).
```

Comment:
Would be good to introduce the `yolo_face` node clearly - I feel that it's introduction in the last sentence was quite sudden.